PocketKit/Sources/PocketKit/MyList/CompactMyListContainerCoordinator.swift

```diff
@@ -128,9 +133,9 @@ class CompactMyListContainerCoordinator: NSObject {
 
         activityVC.completionWithItemsHandler = { [weak self] _, _, _, _ in
             self?.model.archivedItemsList.sharedActivity = nil
-            self?.model.archivedItemsList.selectedReadable?.sharedActivity = nil
+            self?.model.archivedItemsList.selectedItem?.clearSharedActivity()
             self?.model.savedItemsList.sharedActivity = nil
-            self?.model.archivedItemsList.sharedActivity = nil
+            self?.model.savedItemsList.selectedItem?.clearSharedActivity()
```

Comment:
I noticed that this had a duplicate of `self?.model.archivedItemsList.sharedActivity = nil` and seems more logical to replace that with `self?.model.savedItemsList.selectedItem?.clearSharedActivity()`. Let me know if any issues!

Reply:
Nice catch!